Dalamud.FindAnything/Input.cs

```diff
@@ -40,7 +39,7 @@ public static bool IsGameFocused
     public Input()
     {
         try { isTextInputActivePtr = *(IntPtr*)((IntPtr)AtkStage.GetSingleton() + 0x28) + 0x188E; } // Located in AtkInputManager
```

Comment:
I bet this is in CS now, if it isn't we should totally go and add it

Reply:
Looks like this exact code is also used in InventorySearchBar and TPie, so might be useful to have in CS...

I did notice that QoLBar now uses different logic:
```public static bool IsGameTextInputActive => uiModule->GetRaptureAtkModule()->AtkModule.IsTextInputActive();```

Not sure if what differences there might be between these two methods, though.

Reply:
Switched to using QoL Bar's method in the latest commit.